syft/pkg/cataloger/python/package_cataloger.go

```diff
@@ -28,64 +29,34 @@ func (c *PackageCataloger) Name() string {
 }
 
 // Catalog is given an object to resolve file references and content, this function returns any discovered Packages after analyzing python egg and wheel installations.
-func (c *PackageCataloger) Catalog(resolver source.Resolver) ([]pkg.Package, error) {
-	entries, err := c.getPackageEntries(resolver)
-	if err != nil {
-		return nil, err
-	}
+func (c *PackageCataloger) Catalog(resolver source.FileResolver) ([]pkg.Package, error) {
+	// nolint:prealloc
```

Comment:
Can we use a globally toggled-off `prealloc` rule?

Reply:
the original commits are from branches old enough where the prealloc rule was still around (it's not longer there anymore). I'll remove the `nolint` here